WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -657,11 +663,24 @@ - (SiteIconPickerPresenter *)siteIconPickerPresenter
 #pragma mark - iOS 10 bottom padding
 
 - (CGFloat)tableView:(UITableView *)tableView heightForFooterInSection:(NSInteger)section {
+
+    if (self.shouldShowBloggingRemindersCard && section == 0) {
```

Comment:
Sure, I did not think of using that since all the other existing conditions only compare to section indexes. Also, for the section below, I'd still leave the index since we don't want to assume that the section right below it's always gonna be of one kind.